hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -210,28 +336,49 @@ def _check_start_end_dates(self):
                     _('The start date cannot be later than the end date.'))
 
     @api.multi
-    def _get_overlapping_sheet_domain(self, user_id=False):
+    def _get_complete_name_components(self):
```

Comment:
@mreficent @astirpe to give you better insights: this method has another 4 lines of code specific to validation models.